frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts

```diff
@@ -225,8 +225,12 @@ describe('Test clone VM.', () => {
 
     it('Cloned VM has vm.kubevirt.io/name label.', () => {
       expect(
-        searchYAML(`vm.kubevirt.io/name: ${vm.name}`, clonedVM.name, clonedVM.namespace, 'vm'),
-      ).toBeTruthy();
+        _.get(
+          getLabels(getResourceObject(vm.name, vm.namespace, 'vmi')),
+          'vm.kubevirt.io/name',
+          undefined,
```

Comment:
No need to pass the third arg here. If no default is passed, `undefined` would be returned anyway.

Reply:
I wanted to be explicit, but yeah, it's unnecessary. Removed it.